modules/core/02-client/keeper/client.go

```diff
@@ -98,11 +95,8 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.C
 	// Else the update was proof of misbehaviour and we must emit appropriate misbehaviour events.
 	if status := newClientState.Status(ctx, clientStore, k.cdc); status != exported.Frozen {
 		// if update is not misbehaviour then update the consensus state
-		// we don't set consensus state for localhost client
-		if header != nil && clientID != exported.Localhost {
+		if header != nil {
```

Comment:
remove if statement, header was only nil for localhost

Reply:
Done